include/benchmark/benchmark_api.h

```diff
@@ -414,16 +416,18 @@ class State {
   // Range arguments for this run. CHECKs if the argument has been set.
   BENCHMARK_ALWAYS_INLINE
   int range_x() const {
```

Comment:
@dominichamon Do we have any tag for documentation, like doxygen's https://www.stack.nl/~dimitri/doxygen/manual/commands.html#cmddeprecated ?